From 48feb66990f0603ab8d544eac52f26f52d236a2f Mon Sep 17 00:00:00 2001
From: Leon Anavi <leon.anavi@konsulko.com>
Date: Fri, 1 Jul 2016 12:59:10 +0300
Subject: [PATCH] Change resolution to 640x480

Modify the resolution of the home screen to
fit better for QEMU.

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>
---
 Main.qml                             |  4 ++--
 Rotated.qml                          |  4 ++--
 Scaled.qml                           |  4 ++--
 Simple-compositor/main.cpp           |  2 +-
 apps/Dashboard/TireInfo.qml          |  2 +-
 apps/Home/Helix.qml                  |  8 ++++----
 apps/Home/HelixIcon.qml              |  6 ++++--
 apps/Home/Home.qml                   | 16 ++++++++--------
 apps/MediaPlayer/MediaPlayer.qml     |  2 +-
 apps/WaylandView/WaylandView.qml     |  2 +-
 scripts/demohmi.service              |  4 ++--
 start_CES2016_ivi-shell.sh           |  4 ++--
 start_CES2016_with_navi_ivi-shell.sh |  4 ++--
 13 files changed, 32 insertions(+), 30 deletions(-)

diff --git a/Main.qml b/Main.qml
index e6a8160..5cc9829 100644
--- a/Main.qml
+++ b/Main.qml
@@ -23,8 +23,8 @@ import "apps/WaylandView"
 import system 1.0
 
 Item {
-    width: 1080
-    height: 1920
+    width: 640
+    height: 480
 
     Image {
         anchors.centerIn: parent
diff --git a/Rotated.qml b/Rotated.qml
index 4eef7f3..1f40e1b 100644
--- a/Rotated.qml
+++ b/Rotated.qml
@@ -7,8 +7,8 @@
 import QtQuick 2.0
 
 Item {
-    width: 1920
-    height: 1080
+    width: 480
+    height: 640
 
     Main {
         anchors.centerIn: parent
diff --git a/Scaled.qml b/Scaled.qml
index c566735..a0904c6 100644
--- a/Scaled.qml
+++ b/Scaled.qml
@@ -7,8 +7,8 @@
 import QtQuick 2.0
 
 Item {
-    width: 1080*scale
-    height: 1920*scale
+    width: 640*scale
+    height: 480*scale
     scale: 0.5
 
     Main {
diff --git a/Simple-compositor/main.cpp b/Simple-compositor/main.cpp
index 9811bcc..83ebf41 100644
--- a/Simple-compositor/main.cpp
+++ b/Simple-compositor/main.cpp
@@ -45,7 +45,7 @@ int main(int argc, char *argv[])
 
     compositor.setSource(QUrl::fromLocalFile(qmlFile));
     compositor.setTitle(QLatin1String("Simple Compositor"));
-    compositor.setGeometry(0, 0, 1920, 1080);
+    compositor.setGeometry(0, 0, 480, 640);
     compositor.show();
 
     return app.exec();
diff --git a/apps/Dashboard/TireInfo.qml b/apps/Dashboard/TireInfo.qml
index f928c89..9c2fb16 100644
--- a/apps/Dashboard/TireInfo.qml
+++ b/apps/Dashboard/TireInfo.qml
@@ -9,7 +9,7 @@ import "models"
 
 Item {
     id: vehicle_container
-    width: 1080
+    width: 640
     height: 912
     property alias vehicleImage: vehicle_image.source
 
diff --git a/apps/Home/Helix.qml b/apps/Home/Helix.qml
index b47f42c..0413860 100644
--- a/apps/Home/Helix.qml
+++ b/apps/Home/Helix.qml
@@ -61,7 +61,7 @@ Item {
         ListElement { icon: "googlemaps" }
         ListElement { icon: "browser" }
         ListElement { icon: "cameras" }
-        ListElement { icon: "dashboard" }
+        /*ListElement { icon: "dashboard" }
         ListElement { icon: "nfc" }
         ListElement { icon: "media_player" }
         ListElement { icon: "fmradio" }
@@ -70,7 +70,7 @@ Item {
         ListElement { icon: "news" }
         ListElement { icon: "hvac" }
         ListElement { icon: "media_player" }
-        ListElement { icon: "browser" }
+        ListElement { icon: "browser" }*/
     }
 
     Item {
@@ -204,7 +204,7 @@ Item {
             var ydiff = 40 * (h2.cos - h.cos) + spacing
             var atan = Math.atan2(xdiff, ydiff)
             var w = Math.sqrt(xdiff*xdiff + ydiff*ydiff)
-            strand.x = 1080 * 0.5 + xval
+            strand.x = 640 * 0.5 + xval
             strand.y = (row - 1.35) * spacing + 40 * h.cos
             strand.rotation = atan * toDeg
             strand.height = w + 3
@@ -213,7 +213,7 @@ Item {
             ydiff = 40 * (h.cos - h2.cos) + spacing
             atan = Math.atan2(xdiff, ydiff)
             w = Math.sqrt(xdiff*xdiff + ydiff*ydiff)
-            strand.x = 1080 * 0.5 - xval
+            strand.x = 640 * 0.5 - xval
             strand.y = (row - 1.35) * spacing - 40 * h.cos
             strand.rotation = -atan * toDeg
             strand.height = w + 3
diff --git a/apps/Home/HelixIcon.qml b/apps/Home/HelixIcon.qml
index cf34ccb..260d393 100644
--- a/apps/Home/HelixIcon.qml
+++ b/apps/Home/HelixIcon.qml
@@ -20,6 +20,8 @@ Item {
     signal clicked()
 
     Image {
+        sourceSize.width: 100
+        sourceSize.height: 100
         id: background
         x: -0.5 * width
         y: -0.5 * height
@@ -32,8 +34,8 @@ Item {
         anchors.horizontalCenterOffset: 2.5
         anchors.verticalCenterOffset: 2.5
         visible: xscale > 0
-        sourceSize.width: 300
-        sourceSize.height: 300
+        sourceSize.width: 100
+        sourceSize.height: 100
         scale: 0.5
         source: "../../images/" + name + "_app_icon.svg"
     }
diff --git a/apps/Home/Home.qml b/apps/Home/Home.qml
index ad17da9..c16a1c0 100644
--- a/apps/Home/Home.qml
+++ b/apps/Home/Home.qml
@@ -40,7 +40,7 @@ App {
                     color: "white"
                     font.family: "Source Sans Pro"
                     font.weight: Font.Bold
-                    font.pixelSize: 22
+                    font.pixelSize: 10
                     text: "Fuel"
                 }
 
@@ -48,7 +48,7 @@ App {
                     color: "white"
                     font.family: "Source Sans Pro"
                     font.weight: Font.Bold
-                    font.pixelSize: 45
+                    font.pixelSize: 20
                     text: FuelModel.percentage.toFixed(0) + "%"
                 }
             }
@@ -62,7 +62,7 @@ App {
                     color: "white"
                     font.family: "Source Sans Pro"
                     font.weight: Font.Bold
-                    font.pixelSize: 45
+                    font.pixelSize: 20
                     text: TemperatureModel.outdoorString
                 }
 
@@ -71,7 +71,7 @@ App {
                     color: "white"
                     font.family: "Source Sans Pro"
                     font.weight: Font.Bold
-                    font.pixelSize: 22
+                    font.pixelSize: 10
                     text: "Portland OR"
                 }
             }
@@ -90,7 +90,7 @@ App {
                     color: Style.blueViv
                     font.family: "Source Sans Pro"
                     font.weight: Font.Bold
-                    font.pixelSize: 75
+                    font.pixelSize: 36
                     text: SpeedModel.textSpeed.toFixed(0)
                 }
 
@@ -99,7 +99,7 @@ App {
                     color: Style.blueViv
                     font.family: "Source Sans Pro"
                     font.weight: Font.Bold
-                    font.pixelSize: 22
+                    font.pixelSize: 10
                     text: SpeedModel.metric ? "KM/H" : "MPH"
                 }
             }
@@ -113,7 +113,7 @@ App {
                     color: Style.blueViv
                     font.family: "Source Sans Pro"
                     font.weight: Font.Bold
-                    font.pixelSize: 22
+                    font.pixelSize: 10
                     text: "Int. Temp"
                 }
 
@@ -122,7 +122,7 @@ App {
                     color: Style.blueViv
                     font.family: "Source Sans Pro"
                     font.weight: Font.Bold
-                    font.pixelSize: 45
+                    font.pixelSize: 20
                     text: TemperatureModel.indoorString
                 }
             }
diff --git a/apps/MediaPlayer/MediaPlayer.qml b/apps/MediaPlayer/MediaPlayer.qml
index d5b467c..9b61a09 100755
--- a/apps/MediaPlayer/MediaPlayer.qml
+++ b/apps/MediaPlayer/MediaPlayer.qml
@@ -10,7 +10,7 @@ import system 1.0
 
 App {
     appId: "media_player"
-    width: 1080
+    width: 640
 
     Column {
         id: col
diff --git a/apps/WaylandView/WaylandView.qml b/apps/WaylandView/WaylandView.qml
index 6ecc883..e0c460b 100644
--- a/apps/WaylandView/WaylandView.qml
+++ b/apps/WaylandView/WaylandView.qml
@@ -43,7 +43,7 @@ App {
          * is automatically drawn in this item */
         Item {
             id: waylandView
-            // 1920 - TopBar (120) - BottomBar (120) - 100
+            // 480 - TopBar (120) - BottomBar (120) - 100
             height: 1580
             width: parent.width
             Item {
diff --git a/scripts/demohmi.service b/scripts/demohmi.service
index 07838f0..1d7a075 100644
--- a/scripts/demohmi.service
+++ b/scripts/demohmi.service
@@ -6,9 +6,9 @@ After=dbus.service lightmediascanner.service rygel.service
 Type=idle
 Environment=LD_PRELOAD=/usr/lib/libEGL.so
 Environment=SCREEN_W=1080
-Environment=SCREEN_H=1920
+Environment=SCREEN_H=480
 Environment=QT_W=1080
-Environment=QT_H=1920
+Environment=QT_H=480
 Environment=WIN_TITLE_H=30
 ExecStart=/bin/sh -c "sleep 2; /usr/bin/qt5/qmlscene -I /opt/AGL/CES2016/imports /opt/AGL/CES2016/Main.qml"
 ExecStartPost=/bin/sh -c "sleep 3;layer-add-surfaces 1000 1;/usr/bin/LayerManagerControl set surface 2147483648 destination region 0 0 $SCREEN_W $SCREEN_H; /usr/bin/LayerManagerControl set surface 2147483648 source region 0 $WIN_TITLE_H $QT_W $QT_H;"
diff --git a/start_CES2016_ivi-shell.sh b/start_CES2016_ivi-shell.sh
index 542d839..d115c28 100755
--- a/start_CES2016_ivi-shell.sh
+++ b/start_CES2016_ivi-shell.sh
@@ -5,11 +5,11 @@ export AGL_NAVI
 
 #the following value shall be modified for your display side
 SCREEN_W=1080
-SCREEN_H=1920
+SCREEN_H=480
 
 #Demo is configured to FullHD
 QT_W=1080
-QT_H=1920
+QT_H=480
 
 #To cut off window title bar.
 WIN_TITLE_H=30
diff --git a/start_CES2016_with_navi_ivi-shell.sh b/start_CES2016_with_navi_ivi-shell.sh
index 3e71184..af73928 100755
--- a/start_CES2016_with_navi_ivi-shell.sh
+++ b/start_CES2016_with_navi_ivi-shell.sh
@@ -5,11 +5,11 @@ export AGL_NAVI
 
 #the following value shall be modified for your display resolution
 SCREEN_W=1080
-SCREEN_H=1920
+SCREEN_H=480
 
 #Demo is configured to FullHD
 QT_W=1080
-QT_H=1920
+QT_H=480
 
 # surface IDs. This is automatically assigned by wl-shell-emulator.so
 # from base 2147483648(0x80000000) and incremented.
-- 
2.7.4

